style.css

```diff
@@ -0,0 +1,718 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+/* ------------------navbar section------------------- */
+.nav-container {
+  display: flex;
+  width: 1440px;
+  height: 72px;
+  background-color: #3e3c3c;
+  align-items: center;
+}
+
+.nav-container a {
+  display: flex;
+  align-items: center;
+  margin-left: 20px;
+  flex: 1;
+  height: 100%;
+  color: white;
+  text-decoration: none;
+  list-style: none;
+}
+
+.nav-container .items-menu {
+  display: flex;
+  flex: 2;
+  justify-content: center;
+  padding-left: 200px;
+  height: 100%;
+  align-items: center;
+}
+
+.nav-container .items-menu a {
+  margin-right: 40px;
+  color: white;
+  list-style: none;
+  text-decoration: none;
+  font-size: 18px;
+  height: 100%;
+  padding-left: 10px;
+  padding-right: 10px;
+  transition: ease-in-out 5s;
+}
+
+.nav-container .items-menu a:hover {
+  background-color: #615f5f;
+}
+
+.nav-container a li .fa-bars {
+  display: none;
+}
+
+/* ---------------------------------------hero section------------------------------------ */
+.hero-section .hero-row {
+  display: flex;
+  background-color: #1c1a19;
+  width: 1440px;
+  height: 500px;
+}
+
+.hero-left {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  margin-left: 30px;
+  margin-top: 100px;
+  flex: 0.2;
+  height: 100px;
+  transition: transform 5s;
+  order: 1;
+}
+
```

Comment:
- [ ] Kindly make sure to align correctly your social media links in order to meet as much as possible the provided Figma design.

     - ### YOUR DESIGN
     ![image](https://user-images.githubusercontent.com/87197412/212620304-375777e1-c014-4b84-a116-d4d8bdbb70a4.png)

     - ### FIGMA DESIGN
     
     ![image](https://user-images.githubusercontent.com/87197412/212620365-2a49c030-4197-443e-9f1d-664d29eaa801.png)